Module '"ember-concurrency"' has no exported member 'lastValue'

I'm using typescript in ember.
And when following the 2.0.0 upgrade guide, I replace ember-concurrency-decorators imports with ember-concurrency.
```diff
- import { lastValue } from 'ember-concurrency-decorators';
+ import { lastValue } from 'ember-concurrency';
```
I got:
```
error TS2305: Module '"ember-concurrency"' has no exported member 'lastValue'.
```

--- a/addon/index.d.ts
+++ b/addon/index.d.ts
@@ -1391,3 +1391,30 @@ export function waitForProperty<O extends object, K extends keyof O>(
  * ```
  */
 export function forever(): Yieldable<never>;
+
+/**
+ * This decorator allows you to alias a property to the result of a task.
+ * You can also provide a default value to use before the task has completed.
+ *
+ * ```js
+ * import Component from '@glimmer/component';
+ * import { task, lastValue } from 'ember-concurrency';
+ *
+ * export default class ExampleComponent extends Component {
+ *   @task
+ *   someTask = function*() {
+ *     // ...
+ *   };
+ *
+ *   @lastValue('someTask')
+ *   someTaskValue;
+ *
+ *   @lastValue('someTask')
+ *   someTaskValueWithDefault = 'A default value';
+ * }
+ * ```
+ *
+ * @function
+ * @param {string} taskName the name of the task to read a value from
+ */
+export function lastValue(taskName: string): PropertyDecorator;

--- a/tests/types/ember-concurrency-test.ts
+++ b/tests/types/ember-concurrency-test.ts
@@ -37,7 +37,8 @@ import {
   timeout,
   waitForEvent,
   waitForProperty,
-  waitForQueue
+  waitForQueue,
+  lastValue
 } from 'ember-concurrency';
 import { taskFor } from 'ember-concurrency-ts';
 import { expectTypeOf as expect } from 'expect-type';
@@ -2394,6 +2395,8 @@ module('integration tests', () => {
       @task({ maxConcurrency: 1 }) maxConcurrency = function*() {};
       @task({ group: 'foo' }) group = function*() {};
 
+      @lastValue('myTask') myTaskValue = 'or some default';
+
       @restartableTask *myTask(immediately: boolean, ms: number = 500): TaskGenerator<string> {
         // We want to assert `this` is not implicitly `any`, but due `this`
         // being a weird internal type in here, neither of the following
@@ -2453,6 +2456,10 @@ module('integration tests', () => {
         expect(result).not.toBeAny();
         expect(result).toBeString();
         expect(result.length).toBeNumber();
+
+        expect(this.myTaskValue).not.toBeAny();
+        expect(this.myTaskValue).toBeString();
+        expect(this.myTaskValue.length).toBeNumber();
 
         // @ts-expect-error
         myTask.perform('nope');

